thirtyinch-rx/build.gradle

```diff
@@ -26,9 +26,9 @@ dependencies {
     implementation project(':thirtyinch')
     api 'io.reactivex:rxjava:1.3.2'
     implementation 'com.artemzin.rxjava:proguard-rules:1.2.7.0'
-    compileOnly "com.android.support:support-annotations:$supportLibraryVersion"
+    compileOnly 'androidx.annotation:annotation:1.0.0'
```

Comment:
why hardcoding it instead of using the property like we did before?
`$androidXLibrarayVersion`